x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/mapper/DenseVectorFieldMapper.java

```diff
@@ -126,6 +126,7 @@ public DocValueFormat docValueFormat(String format, ZoneId timeZone) {
 
         @Override
         public Query existsQuery(QueryShardContext context) {
+            //TODO how can this work if this field never has doc_values?
```

Comment:
The vector is stored in a BinaryDocValuesField, so I think this is fine.

Reply:
ok but then shouldn't it pass doc_values set to true to the super class?

Reply:
It looks as though `hasDocValues()` is used exclusively for a) indexing (which should go away once everything is properly parametrized) and b) building `exist` queries; it's separate from `isAggregatable` which is used for field caps. So yes, the vector fields should probably set `doc_values` to true in the constructor.

Reply:
I opened #62631 to fix this and add tests for it